asaps/models.py

```diff
@@ -13,24 +11,16 @@
 Field = partial(attr.ib, default=None)
 
 
-class Client:
-    def __init__(self, secfile):
-        """Select secrets.py file for the appropriate instance."""
-        secfile_exists = importlib.util.find_spec(secfile)
-        if secfile_exists is not None:
-            secrets = __import__(secfile)
-        else:
-            secrets = __import__('secretsDocker')
-        print('Editing ' + secfile + ' ' + secrets.baseURL)
-        authclient = ASnakeClient(baseurl=secrets.baseURL,
-                                  username=secrets.user,
-                                  password=secrets.password)
-        authclient.authorize()
-        self.authclient = authclient
+class AsOperations:
+    def __init__(self, client):
+        """Create instance and import client as attribute."""
+        self.client = client
+        start_time = time.time()
```

Comment:
We should keep any time tracking functionality out of this class and address this in another way. We want to make sure our classes are focused on doing one job. This class's job is to passing records to and from the repo.

I'd just leave the time stuff for later, and we'll probably want to think some more about what we want to accomplish with it.

Reply:
Got it